src/pages/OwnerPage/OwnerPage.jsx

```diff
@@ -39,6 +43,24 @@ function OwnerPage() {
   })
 
   useSentryTokenRedirect({ ownerData })
+  const userStartedTrial = localStorage.getItem(
+    LOCAL_STORAGE_USER_STARTED_TRIAL_KEY
+  )
+
+  useEffect(() => {
+    if (userStartedTrial) {
+      renderToast({
+        type: 'generic',
+        title: '14 day trial has started',
+        content: '',
+        options: {
+          duration: 5000,
+          position: 'bottom-left',
+        },
+      })
+      localStorage.removeItem(LOCAL_STORAGE_USER_STARTED_TRIAL_KEY)
+    }
+  }, [userStartedTrial])
```

Comment:
Just an aside, do we really need a useEffect here? What specifically are we using the features of useEffect for that we can't get with just the `if` statement?

Reply:
The toast was rendering multiple times on load. With `useEffect`, we can make sure it only runs once after the initial render.

Reply:
Could you not assign the value accessed from local storage with `let` then set the value to false in your conditional right after the first render?

Reply:
I get mixed behaviour without `useEffect. Either I get the toast rendering many times, or not at all.

Reply:
Grr okay